components/navigation/searchGrid.tsx

```diff
@@ -0,0 +1,28 @@
+import { Input } from '@mui/material';
+
+export const SearchGrid = () => {
+  return (
+    <div className="grid grid-flow-row grid-rows-1 grid-cols-3  bg-primary text-light">
+      <div className="w-full p-4">
+        <Input
+          placeholder="+ Add Courses"
+          className="w-full text-light"
+        ></Input>
+      </div>
+      <div className="w-full p-4">
+        <Input
+          placeholder="+ Add Courses"
+          className="w-full text-light"
+        ></Input>
+      </div>
+      <div className="w-full p-4">
+        <Input
+          placeholder="+ Add Courses"
+          className="w-full text-light"
+        ></Input>
+      </div>
+    </div>
+  );
+};
+
+export default SearchGrid;
```

Comment:
This is just to confirm, do we want there to be three boxes there at all times, just with either a search bar or the search term there, or do we want a single box for the autocomplete and add another box for each term added?

Reply:
If you check the notes that I have at the top of the PR I made reference to the fact that the Search Grid Component in this branch was a placeholder, in all likelihood this file is gonna be deleted once we move the component you've been working on into the dashboard, so I wouldn't worry about this